README.md

```diff
@@ -140,7 +140,7 @@ Breaking down that command:
 
 ## Payments
 
-The connector will facilitate an interledger payment upon receiving a notification for a transfer in which it is credited. That "source" transfer must have a `ilp_header` in its credit's `memo` that specifies the payment's destination and amount.
+The connector will facilitate an interledger payment upon receiving a notification for a transfer in which it is credited. That "source" transfer must have an `IlpHeader` in its credit's `memo.ilp` that specifies the payment's destination and amount.
```

Comment:
I was referring to the `[IlpHeader.json](https://github.com/interledgerjs/five-bells-shared/blob/master/schemas/IlpHeader.json)` schema; is the naming not correct?